File: DartLibraryGenerator/model/local_model_entity.dart
import 'package:code_builder/code_builder.dart';

import 'local_entity_declaration.dart';
import 'local_entity_property.dart';

class LocalModelEntity {
  Library entityImplementation;
  LocalEntityDeclaration declaration;
  List<LocalEntityProperty> properties;

  LocalModelEntity(this.entityImplementation, this.declaration, this.properties);
}

File: NetLibraryGenerator/NetLibraryGenerator/Model/LocalModelEntity.cs
using System.CodeDom;

namespace NetLibraryGenerator.Model
{
    public class LocalModelEntity
    {
        public LocalEntityDeclaration Declaration { get; set; }
        public CodeCompileUnit Implementation { get; set; }
        public List<LocalEntityProperty> Properties { get; set; }

        public LocalModelEntity(LocalEntityDeclaration declaration, CodeCompileUnit implementation, List<LocalEntityProperty> properties)
        {
            Declaration = declaration;
            Implementation = implementation;
            Properties = properties;
        }
    }
}